recipe/meta.yaml

```diff
@@ -1,33 +1,43 @@
-{% set version = "2.3.9" %}
+{% set name = "spyder" %}
+{% set version = "3.1.2" %}
+{% set sha256 = "89a98164e0d27ede273c2b50c8c05044add66c55ab733634d972e1cc55448371" %}
 
 package:
-  name: spyder
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn:  spyder-{{ version }}.zip
-  url: https://pypi.io/packages/source/s/spyder/spyder-{{ version }}.zip
-  md5: dd01e07a77123c128ff79ba57b97c1d7
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-  number: 2
-  osx_is_app: True
+    entry_points:
+        - spyder = spyder.app.start:main
+    number: '0'
+    osx_is_app: 'True'
 
 requirements:
   build:
     - python
+    - sphinx
   run:
     - python
-    - pyqt 4.11.*
+    - pyqt
```

Comment:
We need to keep `pyqt` pinned to ensure it will work with the rest of the `conda-forge` stack.